proto/c1/connector/v2/annotation_trait.proto

```diff
@@ -40,6 +40,7 @@ message UserTrait {
     ACCOUNT_TYPE_HUMAN = 1;
     ACCOUNT_TYPE_SERVICE = 2;
     ACCOUNT_TYPE_SYSTEM = 3;
+    ACCOUNT_TYPE_AI_AGENT = 4;
```

Comment:
What's the difference between this and a service account, and how would we know in an API? Is it a separate call? Is it an attribute?

It seems like this would be a trait option on a user, not necessarily its own user type.